src/Spain/Mews.Fiscalizations.Spain.Tests/Basics.cs

```diff
@@ -72,8 +104,20 @@ private async Task<SimplifiedInvoice> SuccessfullyPostInvoice(Client client)
         var responseErrorMessages = response.Success.Get().Invoices.Select(i => i.ErrorMessage).Flatten();
         var errorMessage = String.Join(System.Environment.NewLine, responseErrorMessages);
         Assert.That(response.Success.Get().Result, Is.EqualTo(RegisterResult.Correct), errorMessage);
+    }
 
-        return invoice;
+    private async Task UnsuccessfullyPostInvoice(Client client, SimplifiedInvoice invoice)
+    {
+        var model = SimplifiedInvoicesToSubmit.Create(
+            header: new Header(Issuer, CommunicationType.Registration),
+            invoices: new[] { invoice }
+        ).Success.Get();
+
+        var response = await client.SubmitSimplifiedInvoiceAsync(model);
+
+        var responseErrorMessages = response.Success.Get().Invoices.Select(i => i.ErrorMessage).Flatten();
+        var errorMessage = String.Join(System.Environment.NewLine, responseErrorMessages);
+        Assert.That(response.Success.Get().Result, Is.EqualTo(RegisterResult.AllIncorrect), errorMessage);
```

Comment:
Duplicate code, you can abstract the logic to a method that accepts the expected result